### PR TITLE
Added Text in Json file and Minor Modification on Some Dialog Design on Prepare Menu.

### PR DESCRIPTION
**Pull request by MeSophie**
I added text on Json file and Adjusted the size of  Labels and controls on some Dialogs in Prepare Menu.
@N-thony and @lloyddewit please can you check it?

### CODE REVIEW

**Review by lloyddewit — APPROVED** [review]

**Comment by lloyddewit** [comment]
@MeSophie Thank you, may I merge or do you expect to make further changes?

**Comment by MeSophie** [comment]
> 

@lloyddewit you can merge so that I can proceed in Crowdin.

File: instat/translations/en/r_instat_not_menus.json [hunk 1]
```diff
@@ -3861,4 +3861,179 @@
...
+  "Day in Year (365)": "Day in Year (365)",
+  "Day in Year (366)": "Day in Year (366)",
+  "Minimum RH (%):": "Minimum RH (%):",
+  "Maximum RH (%):": "Maximum RH (%):",
+  "Base Year:": "Base Year:",
+  "Rug colour:": "Rug colour:",
+  "Spline": "Spline",
+  "Time interval:": "Time interval:",
+  "From:": "From:",
+  "Occurren"
```
Comment (lloyddewit) [inline instat/translations/en/r_instat_not_menus.json:4038]: is this valid?

**Review by lloyddewit — CHANGES_REQUESTED** [review]

**Review by lloyddewit — APPROVED** [review]

File: instat/translations/en/r_instat_not_menus.json (hunk 1 above)
Comment (MeSophie) [reply instat/translations/en/r_instat_not_menus.json]: > is this valid?

@lloyddewit No sorry it was a mistake I corrected it.

**Pull request merged**